docs/crafted-speedbar.org

```diff
@@ -0,0 +1,66 @@
+* Crafted Emacs Speedbar Module
+
+** Installation
+
+To use this module, simply require them in your =init.el= at the appropriate
+points.
+
+#+begin_src emacs-lisp
+;; No additional packages are installed by this module.
+
+;; install the packages
+(package-install-selected-packages :noconfirm)
+
+;; Load crafted-speedbar configuration
+(require 'crafted-speedbar-config)
+#+end_src
+
+** Description
+
+Speedbar is a tree-like view model built-in to Emacs.
+It can be used for viewing file-trees, currently open buffers and more.
+
+- =speedbar-update-flag=: =t=
+
+  Update the speedbar view if the directory of the attached frame changes.
+
+  This behaviour can be changed temporarily by invoking ~speedbar-toggle-updates~.
+  Alternatively, in your configuration you can set it to ~nil~:
+
+  #+begin_src emacs-lisp
+  (setq-default speedbar-update-flag nil)
+  #+end_src
+
+- =speedbar-use-images=: =nil=
+
+  Speedbar uses small images for directories, files and file-local structures,
+  however in many configurations they turn out pixelated or don't fit the theme.
+  Instead, use text versions.
+
+  To set it back to using images, run:
+
+  #+begin_src emacs-lisp
+  (setq-default speedbar-use-images t)
+  #+end_src
+
+- =speedbar-frame-parameters=
+
+  Customizing =speedbar-frame-parameters= sets additional frame parameters
+  for speedbar frames.
+
+  Crafted Emacs adds a frame name/title, removes the minibuffer status
+  and adds some border and fringe to the frame.
+
+- =new keybinding=: =speedbar-switch-to-quick-buffers= (=b=)
+
+  Temporarily change speedbar into a buffer switching tool,
+  displaying the currently open buffers.
+  Pressing =<enter>= on a buffer in the list will switch to that buffer.
+
+- =speedbar-add-supported-extension=: new extensions
+
+  Crafted Emacs adds a wide selection of file extensions that support
+  tagging to speedbar's tagging system.
+  This allows speedbar to add tags when expanding a file entry with this
+  extension (e.g. ~org-mode~ headings) and make them clickable navigation
+  links.
```

Comment:
Here I'm not sure whether an example of how to change this is in order. The Emacs default is included in our settings and more doesn't hurt anybody. So it doesn't really make sense to put an example here. Aesthetically, I have the feeling that something's missing :shrug:

Reply:
Don't really see what do add here, I'll resolve the rest, if you have any suggestions, I can add them

Reply:
You're right. I have no idea, either. 'Twas but a thought. Nevermind.